relayer/scheduler.go

```diff
@@ -190,13 +236,40 @@ func (s *Scheduler) SyncTunnels(ctx context.Context) {
 			chainProvider,
 		)
 
+		// update metrics for the new tunnel and its target chain status
+		t, err := tr.TargetChainProvider.QueryTunnelInfo(ctx, tr.TunnelID, tr.ContractAddress)
+		if err != nil {
+			continue
+		}
+
+		if relayermetrics.IsTelemetryEnabled() {
+			tr.IsTargetChainActive = t.IsActive
+			status := targetContractActiveStatus
+			if !t.IsActive {
+				status = targetContractActiveStatus
+			}
+			relayermetrics.IncTargetContractCount(status)
+
+			if _, ok := s.ChainNames[tunnels[i].TargetChainID]; !ok {
+				s.ChainNames[tunnels[i].TargetChainID] = true
+			}
+		}
+
 		s.TunnelRelayers = append(s.TunnelRelayers, &tr)
 		s.isErrorOnHolds = append(s.isErrorOnHolds, false)
+
 		s.Log.Info(
 			"New tunnel synchronized successfully",
 			zap.String("chain_name", tunnels[i].TargetChainID),
 			zap.Uint64("tunnel_id", tunnels[i].ID),
 		)
+		s.BandLatestTunnel = int(tunnels[i].ID)
```

Comment:
if we have new 5 tunnels (ID 1-5), and tunnelID 2,3, and 5 fails due to rpc error, should we retry creating tunnelRelayer for ID 2 and 3 (ID 5 will be created next time if the rpc is back)? @RogerKSI